source/server-discovery-and-monitoring/tests/monitoring/standalone_debounce_description_updates.yml

```diff
@@ -0,0 +1,73 @@
+description: "Monitoring a standalone connection - multiple equal description updates are debounced"
```

Comment:
I hadn't heard of "debouncing" before this PR, but it seems to pertain to ensuring events don't fire too often (often in the context of UI). This is PR is about avoiding emission of a redundant event entirely because there is no change to report. Would it make more sense to say "suppress update events for unchanged server descriptions" or something to that effect?

Reply:
I like Jeremy's wording.

Reply:
Updated file names and descriptions, PTAL